docs/generate-time-series/javascript.rst

```diff
@@ -0,0 +1,232 @@
+.. _gen-ts-javascript:
+
+==========================================
+Generate time series data using Javascript
+==========================================
+
+This tutorial will show you how to generate some :ref:`experimental time series
+data <gen-ts>` by sampling your `system load`_ using Javascript, specifically `Node.js`_.
+
+.. SEEALSO::
+
+    :ref:`gen-ts`
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+Prerequisites
+=============
+
+CrateDB must be :ref:`installed and running <install-run>`.
+
+Make sure you're running up-to-date Node/npm.
+
+The `node-postgres`_ driver is our officially supported driver for Javascript.
+With this in mind, use `npm`_ to install the `pg`_ library:
```

Comment:
I think we can probably drop the big about officially supporting it. that can probably be mentioned on the list of supported clients page. in this context, it's implicit, I think

how about something like this:

```
Use npm to install the `node-postgres`_ library:

[...]
```

here, I've used `node-postgres` in full in the prose, because it's easier to read and understand, but then switched to `pg` in the command, which hopefully is enough of an explanation for the reader that these two things are synonyms

thoughts?

Reply:
I think this makes sense, aye.